spec/rubocop/cop/style/string_literals_spec.rb

```diff
@@ -194,6 +194,14 @@
       expect(cop.config_to_allow_offenses).to eq('Enabled' => false)
     end
 
+    it 'registers offense for escaped single quote in single quotes' do
```

Comment:
Added. The new test case name differs because with "registers offense" the line would be 81 char :stuck_out_tongue: